polytiles/polytiles.py

```diff
@@ -271,18 +267,16 @@ def multithreading(self):
         return True
 
     def close(self):
-        self.p_pipe.send(('close', None))
-        self.join()
+        self.queue.put(("close", None))
+        super().join()
 
 
-def multi_MBTilesWriter(threads, filename, setname, overlay=False,
-                        version=1, description=None, format='png256'):
-    params = {'filename': filename, 'setname': setname, 'overlay': overlay,
-              'version': version, 'description': description, 'format': format}
+def multi_MBTilesWriter(threads, filename, setname, overlay=False, version=1, description=None, format="png256"):
```

Comment:
Again, line lengths here and below. Don't forget to check with a PEP8 validator!